### PR TITLE
SoundWire: remove obsolete acpi configurations

### PR DESCRIPTION
**Pull request by plbossart**
We've removed the topologies used by those two configurations so let's proceed with the kernel cleanup.

### CODE REVIEW

File: sound/soc/intel/common/soc-acpi-intel-tgl-match.c [hunk 1]
```diff
@@ -182,15 +182,6 @@ static const struct snd_soc_acpi_adr_device rt714_3_adr[] = {
 	}
 };
 
-static const struct snd_soc_acpi_link_adr tgl_i2s_rt1308[] = {
```
Comment (ranj063) [inline sound/soc/intel/common/soc-acpi-intel-tgl-match.c:185]: @plbossart duplicate sign-off in commit message and did you mean RT711 in SDW mode and RT1308 in I2S mode?

Comment (plbossart) [reply sound/soc/intel/common/soc-acpi-intel-tgl-match.c:185]: yes, thanks. will fix.

**Review by kv2019i — APPROVED** [review]

**Comment by plbossart** [comment]
@bardliao @RanderWang can you please take a look?

**Review by bardliao — APPROVED** [review]
LGTM

**Review by RanderWang — APPROVED** [review]

**Pull request merged**